docs/db/ER-diagram-text.txt

```diff
@@ -7,6 +7,8 @@ Table "tenant" {
   "id" "text" [pk, not null]
   "name" "text" [not null, unique]
   "consortium_id" uuid [not null]
+  "code" "text"
+  "is_central" bool
```

Comment:
if another table exists that holds the central id for a consortium, is it still needed to have a `is_central` column on the tenants table?

Reply:
It will help that we can identify tenants whether central or not, when creating tenants. without this column, we don't know that tenant should be central or not.
![image](https://github.com/folio-org/mod-consortia/assets/113523904/2d7a8089-8d71-46cf-9d1e-6daf175c533c)

Reply:
I see your point, basically is_central is a key property that describes tenant so we can leave it in the table in central tenant, and this new config property will be stored in all tenants(central and institutional).
Also this property can be usefull for API users that retrieving list of tenants or for example during login when user need to see list of tenants and to understand what is central (and avoid additional collecting data together)

Reply:
There is potential to have the isCentral returned in the api but not have a database column for it. The service could populate the isCentral property in the DTO after querying consortia configuration. OR a left join between tenant table and consortia configuration table where the result of a sql will have a derived/calculated column of isCentral. Essentially, one data point will rely on another instead of two separate ones.

I dont feel strongly about this, but wanted to raise the concern.

Reply:
okay if you think we need to rewrite existing approach it please let us know

Reply:
both approaches have pros/cons actually :)
existing one introduces additional column, at the same time it's simpler, as for another one doesn't require column, but it's a bit more complicated to compare with the first one

both solve the problem to provide info about central tenant in affiliation, so we should just agree which one to use

@okolawole-ebsco is your main concern that we'll have a column which value can be populated in runtime?

Reply:
@SerhiiNosko We can go ahead with what we have. Central tenant Id should not mutate too often. If an issue comes up in the future, we know how to fix it.

@NikitaSedyx My main concern was about keeping the same information in sync with both places. it could be possible to have /consortia-configuration set to `tenantA` and isCentral in the tenant table set to `tenantB`. More code would be written to keep them synchronized.

Reply:
@okolawole-ebsco  makes sense, especially if it will be a case to change central tenant